src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -39,6 +40,7 @@ internal sealed partial class Http2Connection : HttpConnectionBase, IDisposable
         private bool _expectingSettingsAck;
         private int _initialWindowSize;
         private int _maxConcurrentStreams;
+        private DateTimeOffset _lastFinishedStream;
```

Comment:
Can we name this better, e.g. _lastUseTime or similar?

Reply:
Or "_idleSinceTime" might be better.

Reply:
I like _idleSinceTime.